modules/network-observability-operator-install.adoc

```diff
@@ -21,6 +21,13 @@ You can install the Network Observability Operator using the {product-title} web
 +
 * *spec.agent.ebpf.Sampling* : Specify a sampling size for flows. Lower sampling sizes will have higher impact on resource utilization. For more information, see the `FlowCollector` API reference, under spec.agent.ebpf. 
 * *spec.deploymentModel*: If you are using Kafka, verify Kafka is selected.
+* *spec.exporters*: If you are using Kafka, you can optionally send network flows to Kafka, so that they can be consumed by any processor or storage that supports Kafka input, such as Splunk, Elasticsearch, or Fluentd. To do this, set the following specifications:
```

Comment:
@jotak @stleerh PTAL at these new list items. Joel, should I add tls specifications to the YAML example as well? Steven was saying the tls for this option should be the same as with Loki configuration, but Kafka instead of Loki.

Reply:
Yes we can mention how to configure tls, actually better than Loki we can copy how it was written for the existing Kafka config (not as an exporter): this is here: https://docs.openshift.com/container-platform/4.12/networking/network_observability/configuring-operator.html#network-observability-flowcollector-kafka-config_network_observability

The only difference is that the kafka certificates will be necessary only for FLP pods, not for the ebpf agent pods. So the sentence
> [...] certificate must be available as a ConfigMap or a Secret, both in the namespace where the flowlogs-pipeline processor component is deployed (default: netobserv) and where the eBPF agents are deployed (default: netobserv-privileged)

would be:
`[...] certificate must be available as a ConfigMap or a Secret, in the namespace where the flowlogs-pipeline processor component is deployed (default: netobserv).`